packages/react/src/components/limit/Limit.tsx

```diff
@@ -11,15 +11,47 @@ import {InputSelectors} from '../input/InputSelectors';
 import {Svg} from '../svg';
 
 export interface LimitOwnProps {
+    /**
+     * Unique identifier of the Limit card
+     */
     id?: string;
+    /**
+     * The text displayed as the title of the Limit card
+     */
     title: string;
+    /**
+     * The text displayed as the title of the limit value
+     *
+     * @default Limit
+     */
     limitLabel?: string;
+    /**
+     * Additionnal CSS classes to customize the style of the Limit card
+     */
     className?: string;
+    /**
+     * Display the current usage under the Usage label
+     */
     usage?: number;
+    /**
+     * Set the limit value of the Limit card. When the limit is reached, the progress bar is turning red color.
+     */
     limit?: number;
+    /**
+     * Whether or not the limit value is editable directly in the Limit card
```

Comment:
I think we should omit the `or not` everywhere https://www.grammarphobia.com/blog/2013/03/whether-or-not.html

```suggestion
     * Whether the limit value is editable directly in the Limit card
```